src/Database/Bloodhound/Types.hs

```diff
@@ -51,7 +51,11 @@ module Database.Bloodhound.Types
        , omitNulls
        , BH(..)
        , runBH
-       , BHEnv(..)
+       , BHEnv
```

Comment:
Yeah I think this is the setup I wanted with the Internal and non-Internal module. Thank you!